server/src/hardware/protocol/z21/messages.hpp

```diff
@@ -1423,6 +1444,69 @@ constexpr std::string_view toString(LanXSetLocoFunction::SwitchType value)
   return {};
 }
 
+struct MessageReplyType
+{
+  enum Priority
+  {
+    LowPriority = 0,
+    Normal = 1,
+    Urgent = 2
+  };
```

Comment:
Why does one element have `Priority` in it and the other don't have it?

Change it to `enum class Priority : uint8_t`? Then the values are scoped :)